libpod/options.go

```diff
@@ -1816,40 +1813,39 @@ func WithInitCtrType(containerType string) CtrCreateOption {
 // namespace sharing within a pod. Typically, an infra
 // container is lightweight and is there to reap
 // zombie processes within its pid namespace.
-func WithInfraImage(img string) PodCreateOption {
+func WithInfraImage(img string, infraSpec *specgen.SpecGenerator) PodCreateOption {
```

Comment:
This should be set as part of pod creation - shouldn't require the spec be passed in

Reply:
Honestly, as is, this should be entirely removed. Same with `WithInfraCommand`. We should just set the fields in the infra container's config.

Reply:
never even called, deleting.